kubernetes/src/main/java/io/kubernetes/client/ApiClient.java

```diff
@@ -90,6 +90,7 @@ public ApiClient() {
         // Setup authentications (key: authentication name, value: authentication).
         authentications = new HashMap<String, Authentication>();
         authentications.put("BearerToken", new ApiKeyAuth("header", "authorization"));
+        authentications.put("BasicAuth", new HttpBasicAuth());
```

Comment:
Follow up on my previous comment, if we are manually editing ApiClient.java, we should add it to the ignore list and remove it when the upstream PR is merged.